apps/core-app/src/renderer/src/modules/box/adapter/hooks/useClipboardChannel.ts

```
@@ -10,6 +10,9 @@ import {
   getLatest as getLatestClipboardItem,
   onClipboardChange
 } from '../transport/clipboard-transport'
+import { createRendererLogger } from '~/utils/renderer-log'
```

ESLint

[error] Expected "~/utils/renderer-log" (internal) to come before "../transport/clipboard-transport" (parent).

(perfectionist/sort-imports)